lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -72,29 +74,29 @@ local function getWeaponWeight( weapon )
     return 1
 end
 
-local function getWeaponCount( ply ) 
+local function getPlayerWeight( ply ) 
     if playerIsInBuild( ply ) then return 0 end
     local weapons = ply:GetWeapons()
-    local weaponCount = 0
+    local totalWeight = 0
     for _, weapon in pairs( weapons ) do
-        weaponCount = weaponCount + getWeaponWeight( weapon )
+        totalWeight = totalWeight + getWeaponWeight( weapon )
     end
-    return weaponCount
+    return totalWeight
 end
 
 -- Hook Functions --
 local function onEquip( wep, ply )
     if not IsValid( ply ) then return end
-    local weaponCount = getWeaponCount( ply ) + getWeaponWeight( wep )
+    local totalWeight = getPlayerWeight( ply ) + getWeaponWeight( wep )
```

Comment:
Off topic, but is this run before or after the weapon is actually equipped? Will we be counting weapons twice by running `getPlayerWeight` here?

Reply:
before